hasura/metadata/actions.yaml

```diff
@@ -1,15 +1,32 @@
 actions:
+- name: createUser
+  definition:
+    kind: synchronous
+    handler: '{{HASURA_ACTION_BASE_URL}}/create_user'
+    forward_client_headers: true
+    headers:
+    - name: verification_key
+      value_from_env: HASURA_EVENT_SECRET
+  permissions:
+  - role: user
```

Comment:
https://github.com/coordinape/coordinape/pull/508/files#diff-f3e37af54b5b154e8a705d0f72ed476d9b054ed166a0f8fcf25aece0d8ba4e7cR24 seems to only allow this for admins, but we also allow this action for `role: user` too ?

Reply:
Yeah, some users are circle admins, but they don't carry any special privileges above user in hasura, since anyone can become a circle admin simply by creating a circle.

We have a middleware (that I added today) that checks the requesting user is a circle admin for the circle they're attempting to modify, and that's how we're apparently gating this logic after some discussion with @exrhizo @META-DREAMER and @zashton in the dev core channel.